src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -488,6 +491,7 @@ public partial struct Byte : System.IComparable, System.IComparable<byte>, Syste
     }
     public partial struct Char : System.IComparable, System.IComparable<char>, System.IConvertible, System.IEquatable<char>
     {
+        private object _dummy;
```

Comment:
Same as Boolean - why object?

Reply:
Was an issue in detecting a reference type, just fixed.